Owncloud iOs Client/Login/Keychain/OCKeychain.h

```diff
@@ -19,6 +19,9 @@
 @interface OCKeychain : NSObject
 
 +(BOOL)setCredentialsById:(NSString *)idUser withUsername:(NSString *)userName andPassword:(NSString *)password;
```

Comment:
This is the chance to rename the first parameter in all the other methods to ```userId``` instead of ```idUser```.

Reply:
👍 done